amplicon-pooling.ipynb

```diff
@@ -874,7 +874,9 @@
     "         'BarcodesAreRC': 'False',\n",
     "         'ForwardAdapter': '',\n",
     "         'ReverseAdapter': '',\n",
-    "         'HumanFiltering': 'True'\n",
+    "         'HumanFiltering': 'True',\n",
+    "         'library_construction_protocol': '',\n",
+    "         'experiment_design_description': '',\n",
```

Comment:
If we set this as None, or an empty string will the validation fail? If so let's do that and leave a comment that instructs something like fill this value with a short description of the experiment's details.

Reply:
I just committed a change that warns the user if library_construction_protocol or experiment_design_description is an empty string or None, similar to other columns (i.e. I5_Index_ID, Well_description). Should we leave a more descriptive comment for these?